FHEM/lib/signalduino_protocols.hash

```diff
@@ -1696,4 +1696,31 @@
 		length_min    => '12',
 		length_max    => '12',
 	},
+	"87" => # Novy Pureline 6830 kitchen hood
+			# https://github.com/RFD-FHEM/RFFHEM/issues/331
+            # light on/off button
+			# MU;P0=710;P1=353;P2=-403;P4=-761;P6=-16071;D=20204161204120412041204120414141204120202041612041204120412041204141412041202020416120412041204120412041414120412020204161204120412041204120414141204120202041;CP=1;R=40;
+            # + button
+			# MU;P0=22808;P1=-24232;P2=701;P3=-765;P4=357;P5=-15970;P7=-406;D=012345472347234723472347234723454723472347234723472347234547234723472347234723472345472347234723472347234723454723472347234723472347234;CP=4;R=39;
+            # - button
+			# MU;P0=-8032;P1=364;P2=-398;P3=700;P4=-760;P5=-15980;D=0123412341234123412341412351234123412341234123414123512341234123412341234141235123412341234123412341412351234123412341234123414123;CP=1;R=40;
+            # power button
+			# MU;P0=-756;P1=718;P2=354;P3=-395;P4=-16056;D=01020202310231310202 42 310231023102310231020202310231310202 42 31023102310231023102020231023131020242310231023102310231020202310231310202;CP=2;R=41;
+            # novy button
+			# MU;P0=706;P1=-763;P2=370;P3=-405;P4=-15980;D=0123012301230304230123012301230123012303042;CP=2;R=42;
+	{
+		name 		  => 'Novy',
+		comment       => 'Pureline_6830',
+		id            => '87',
+		zero		  => [-2,1],
+		one           => [-1,2],
+		start         => [-44,1],
+		clockabs      => 365,
+		format        => 'twostate',
+		preamble      => 'u87#',	  # prepend to converted message	
+		#clientmodule  => 'SD_UT',    # not used now
+		#modulematch   => '^u87#',    # not used now
```

Comment:
Das Kommentar # not used now verwirrt und sollte entfernt werden, da es so nicht mehr stimmt.
Attribut wird, sofern vorhanden, ausgewertet.

Reply:
Ich habe es entfernt.
Da noch mehr solche Kommentare in der Datei sind, so würde ich diese auch entfernen wollen um es zu vereinheiltlichen in einem neuen PR. Einverstanden?

Reply:
Die Protokolle 86 und 87 sind ja wohl identisch, bis auf den geringen Unterschied bei clockabs. Da sollte ein Eintrag mit Länge 12 bis 18 reichen und im Modul muss der User mittels Attribut sein passendes Model wählen.

Reply:
Meinst du nicht, das der start unterschiedlich ist? Sollte die nachricht mal wieder als ms erkannt werden, so könnten wir beim senden probleme bekommen oder habe ich da umsonst bedenken?

Reply:
Mhmm, ich sehe bei beiden:
start         => [-44,1],
und bei beiden einen Sync von ca. -16000 und 300.